src/test/unit_tests/test_bucketspace.js

```diff
@@ -578,7 +597,7 @@ async function update_account_nsfs_config(email, default_resource, new_nsfs_acco
 mocha.describe('list objects - namespace_fs', function() {
     const nsr = 'nsr1-list';
     const bucket_name = 'bucket-to-list1';
-    const tmp_fs_root = '/tmp/test_bucket_namespace_fs1';
+    let tmp_fs_root = get_tmp_path_by_os('/tmp/test_bucket_namespace_fs1');
```

Comment:
it can still be const no? in all cases...
```suggestion
    const tmp_fs_root = get_tmp_path_by_os('/tmp/test_bucket_namespace_fs1');
```